src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -55,9 +57,6 @@
   /** All the conversations */
   private List <Conversation> conversations;
 
-  /* Only messages that correspond to a given user ID */
-  private List <Message> realMessages = new ArrayList<>();
```

Comment:
Why was this previously an instance variable and why is it now moved into the doGet method as a local variable? Would it make things more efficient if we built an instance variable of a Map that maps Strings (usernames) to a List<Message> (that user's messages)?

Reply:
I noticed when it was an instance variable each time the page was rendered, more messages were just added to realMessages, ending up in a bunch of repeated messages being displayed on the profile. 

Making it a local variable makes sure that each message is only rendered once I think!